tests/integration/tui/screenshots/test_navigation.cursor-at-bottom-of-screen.html

```diff
@@ -0,0 +1,31 @@
+<pre><span style="color:#000000;background:#e5e5e5">                                                                                </span>
```

Comment:
I was really hoping GitHub would render this HTML which would make it much easier to review screenshot updates in PRs. Any ideas?

Reply:
The best thing I can come up with is to use some Python lib to render them into png derived files.